site/docs/v1/tech/apis/_user-response-body.adoc

```diff
@@ -2,6 +2,9 @@
 
 [.api]
```

Comment:
If this is the Login Response definition, it is out of date. Can you sync it up? 
https://github.com/FusionAuth/fusionauth-java-client/blob/39ed46a3380816695d987988f676dc37df51abfe/src/main/java/io/fusionauth/domain/api/LoginResponse.java#L30

Reply:
Done